src/shared/components/Lists/index.tsx

```diff
@@ -45,9 +56,54 @@ const ListsContainer: React.FunctionComponent<ListProps> = React.memo(
             </li>
           );
         })}
-        <div className="list -new" onClick={createList} key="make-new-list">
-          <h2>Make a new list</h2>
-          <p>{'view resources from project '}</p>
+        {!(projectLists || []).length && (
+          <div style={{ opacity: 1, width: '50%', marginTop: '5%' }}>
+            <Empty description>No lists</Empty>
+          </div>
+        )}
+        <div
+          className="side-panel"
+          key="make-new-list"
+          style={{ width: '240px', padding: '0 1em' }}
+        >
+          <h2>Resources</h2>
+          <p>view resources from a project</p>
+          <div style={{ display: 'flex', flexDirection: 'column' }}>
+            <ResourceForm
+              project={project}
+              onSuccess={() => forceUpdate()}
+              render={(updateFormVisible: () => void) => {
+                return (
+                  <Button
+                    style={{ margin: '0.5em 0' }}
+                    type="primary"
+                    onClick={updateFormVisible}
+                    icon="plus-square"
+                  >
+                    Create Resource
+                  </Button>
+                );
+              }}
+            />
+            <Button
+              style={{ margin: '0.5em 0' }}
+              onClick={createList}
+              icon="plus-square"
+            >
+              New Query
+            </Button>
+            <Link to={`/${orgLabel}/${projectLabel}/graph/sparql`}>
```

Comment:
Same goes with this I guess. The button should call a function that comes from your props. Then in your view, you can make it Link somewhere if you want.